src/FactoryTrait.php

```diff
@@ -191,14 +191,13 @@ public function factory($seed, $defaults = [], string $prefix = null): object
      * add prefix.
      *
      * Rules observed, in order:
-     *  - If class starts with "." then prefixing is always done.
-     *  - If class contains "\" prefixing is never done.
+     *  - If class starts with "./" then prefixing is always done.
+     *  - If class contains "\" or it is an anonymous class prefixing is never done.
      *  - If class (with prefix) exists, do prefix.
      *  - don't prefix otherwise.
      *
      * Example: normalizeClassName('User', 'Model') == 'Model\User';
      * Example: normalizeClassName(Test\User::class, 'Model') == 'Test\User'; # or as per "use"
-     * Example: normalizeClassName('Test/User', 'Model') == 'Model\Test\User';
      * Example: normalizeClassName('./User', 'Model') == 'Model\User';
```

Comment:
What if we have `.\User` here?

Reply:
I do not like two variants or relative paths at all. But as long as relative paths are allowed, probably both variants can be allowed.

As the NS separator is `\\`, allow only `.\\` instead of `./`?

Reply:
fixed, good?

Reply:
I know a project which contains hundreds of classnames like `namespace/modelName` :)
But you're right, it's wrong slash there. I was just wondering maybe we still could have `str_replace('/', '\\', $name)` there at the beginning and then it would be fully compatible.
In this particular case I do not see big advantage of being so strict.

Reply:
I think we should enforce the proper way. The replace was breaking at least anonymous classes, later, this prefixing should be completely removed, thus we should enforce the right usage now.

Reply:
well, ok :)